homeassistant/components/myuplink/strings.json

```diff
@@ -40,5 +40,25 @@
         "name": "Status"
       }
     }
+  },
+  "exceptions": {
+    "config_entry_auth_failed": {
+      "message": "Error while logging in to the API."
+    },
+    "config_entry_not_ready": {
+      "message": "Error while loading the integration."
+    },
+    "incorrect_oauth2_scope": {
+      "message": "Incorrect OAuth2 scope."
```

Comment:
Let's not make this too technical, let's make this something like "stored authentication does not have enough permission, asking a refresh". Okay, I am also not happy with this either, but you get the idea

Reply:
This is perhaps better. In any case the old out-dated scope is updated automatically.

Reply:
Well, more or less automatically. The scope will be updated when the user reauthenticates.